internal/pkg/ssm/command.go

```diff
@@ -19,7 +19,7 @@ var initE2EDirCommand = "mkdir -p /home/e2e/bin && cd /home/e2e"
 
 func WaitForSSMReady(session *session.Session, instanceId string) error {
 	err := retrier.Retry(10, 20*time.Second, func() error {
-		return Run(session, instanceId, "ls")
+		return Run(session, logr.Discard(), instanceId, "ls")
```

Comment:
would it be useful to be able to toggle the discard on/off? using an env var or flag perhaps